### ISSUE TITLE
Support multiple versions of clusterctl

### ISSUE THREAD

**Issue by lb4368** [airshipit/airshipctl#650]
**Problem description (if applicable)**
In order to support multiple CAPI providers (capm3, capz, capo, etc) and their associated release schedules, airshipctl needs to support multiple versions of CAPI concurrently.  The current **clusterctl** KRM function only supports a single clusterctl CLI version.

**Proposed change**

- Provide support within airshipctl for multiple versions of clusterctl
- Provide a mechanism to determine the proper version of clusterctl to use for a deployment

**Comment by shon-phand** [comment]
Please assign this to me, I am working on it.

**Comment by mattmceuen** [comment]
There have been discussions around keeping the latest versions as simply "clusterctl", and only using version numbers for exception cases, when a CAPI provider needs to use an old version.  Just want to make sure, @raliev12 are you and Alexey aligned to this?

**Comment by aodinokov** [comment]
>only using version numbers for exception cases, when a CAPI provider needs to use an old version

@mattmceuen yes, we're aligned.. the patchset from Ruslan makes it work that way
